doc/conf.py

```diff
@@ -125,6 +125,8 @@
 #html_style = 'default.css'
 # html_theme = 'sphinxdoc'
 
+html4_writer=True
```

Comment:
Equal sign needs spaces around..

Also, can you add a comment to explain why you need it to change to the html4 writer instead of using the default? (https://www.sphinx-doc.org/en/master/usage/configuration.html#confval-html4_writer)

Reply:
Sounds great, acknowledged! :D